features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -217,6 +217,17 @@
 
   "oauth.global_rbac_scope_issuer.enable": false,
 
+  "oauth.jarm.enable": true,
```

Comment:
do we need a separate one here?

Reply:
This is resolved with the commit ID c6da8c20e46455e080693db31c128ff6839d994d